test/tools/ossfuzz/protoToAbiV2.cpp

```diff
@@ -695,20 +810,34 @@ void ProtoConverter::visit(TestFunction const& _x)
 		if (returnVal != 0)
 			return uint(200000) + returnVal;
 
+		<?atLeastOneVar>
 		bytes memory argumentEncoding = abi.encode(<parameter_names>);
 
-		returnVal = checkEncodedCall(this.coder_public.selector, argumentEncoding, <invalidLengthFuzz>);
+		returnVal = checkEncodedCall(
+			this.coder_public.selector,
+			argumentEncoding,
+			<invalidLengthFuzz>,
+			<isRightPadded>
+		);
 		if (returnVal != 0)
 			return returnVal;
 
-		returnVal = checkEncodedCall(this.coder_external.selector, argumentEncoding, <invalidLengthFuzz>);
+		returnVal = checkEncodedCall(
+			this.coder_external.selector,
+			argumentEncoding,
+			<invalidLengthFuzz>,
+			<isRightPadded>
+		);
 		if (returnVal != 0)
 			return uint(200000) + returnVal;
+		</atLeastOneVar>
 		return 0;
 	}
 	)")
 	("parameter_names", dev::suffixedVariableNameList(s_varNamePrefix, 0, m_varCounter))
```

Comment:
In general, the naming seems to be inconsistent here. Or is this desired?

Reply:
No, it was my oversight. Will rename it to `parameterNames`